content/guides/accessibility-for-teams/ux-design.md

```diff
@@ -0,0 +1,171 @@
+- Take an Accessibility 101 class, such as [Udacity’s Web Accessibility class](https://www.udacity.com/course/web-accessibility--ud891), to get a baseline knowledge of accessibility.
+- Learn how to [navigate a webpage using only your keyboard]({{< link "/guides/accessibility-for-teams/front-end-development/" >}}) and learn how to use a screen reader, [such as VoiceOver on your Mac]({{< link "/guides/accessibility-for-teams/front-end-development/" >}}) so you can spot check new features when necessary. This can help you understand the technology itself, but keep in mind that folks who use it every day will often have their own strategies for using these tools.
+- Observe people using assistive technology (AT) on your product or others. In some cases you may be able to find videos demonstrating how people use different strategies to interact with digital products.
+
+### Resources
+- [1.3 Adaptable (Guideline)](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=128%2C14&currentsidebar=%23col_overview#content-structure-separation)
+- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=128%2C14&currentsidebar=%23col_overview#content-structure-separation-programmatic)
+
+## Humanize accessibility for your team.
+
+- Cover accessibility and inclusive design issues when conducting user research. Consider the native language, [literacy](http://contentsmagazine.com/articles/the-audience-you-didnt-know-you-had/), digital literacy, and digital access of your users as well as potential visual, hearing, motor, and cognitive disabilities.
+- Incorporate accessibility considerations in your personas, user archetypes, or user stories. This could take the form of swapping out [accessibility issues in your personas](https://www.perpendicularangel.com/portfolio/publications/published-on-the-pastry-box/an-alphabet-of-accessibility-issues/) or creating standalone user stories focused on accessibility.
+- Help your team learn about the experience of using your product with assistive technology. (Keep in mind that the way a person with a disability uses assistive tools may be very different from your experience of the tools.) This could include:
+  - Ask each person on your team to turn off their mouse and trackpad and navigate the product using only their keyboard.
+  - Find an appropriate [empathy prompt](https://empathyprompts.net/#diminished-problem-solving-skills) and walk your team through it.
+
+## Tab order
+### Why it's important
+Do your wireframes or design mockups indicate a logical tab order for people using a keyboard, or other assistive technology, to navigate?
+
+- Maria has tendonitis and is unable to use a mouse; instead, she uses the keyboard to navigate the web. When focus jumps randomly around the page she gets confused.
+
+
```

Comment:
@bonnieAcameron I added this text break to separate content between the two lists.

Reply:
@bonnieAcameron Unfortunately adding white space to a bulleted list does not honor the white space. I added some text instead. Please review.